bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -62,7 +60,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
       if (!isRegistered(userService.getCurrentUser().getUserId())
           && !request.getRequestURI().startsWith("/_ah/")) {
         // User is submitting form for registration
```

Comment:
How does this tell us they're submitting the form for registration? Expand upon this comment.

Reply:
Clarified